pkg/model/manifest.go

```diff
@@ -410,12 +410,9 @@ func getManifestFromFile(cwd, manifestPath string) (*Manifest, error) {
 		for _, composeInfo := range stackManifest.Deploy.ComposeSection.ComposesInfo {
 			composeFiles = append(composeFiles, composeInfo.File)
 		}
-		s, stackErr := LoadStack("", composeFiles, false)
-
-		// We failed to load a stack file and a manifest file, so we need to return
-		// only the original manifest error
```

Comment:
According to the code comment, this was done on purpose. Have we checked or asked about why we should be returning the original manifest error?

And why the code was getting a panic? My understanding is that this should be returning an error (not the one we wanted), but as it's an error, I would expect that the execution is broken instead of keep going and get a panic

Reply:
talked offline, we will run e2e tests to see if they pass or not with these changes

Reply:
@teresaromero should we also change it here? https://github.com/okteto/okteto/blob/bf5f3cde626c900e021f3b626c82a952bea2d77c/pkg/model/manifest.go#L440

I think that scenario is when we refer to a docker compose from an okteto manifest V2. If that docler-compose also have a depends_on, it might fail also as we are not validating the stack there.

Reply:
Yes, that is right, i've just debugged this and added validation. In this case there is also a panic if an okteto manifest has a compose section with an invalid compose...

In this case the err returned is this one